documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -15,168 +15,126 @@ MirrorMaker 2 consists of the following connectors:
 
 `MirrorSourceConnector`:: (Required) Replicates topics and ACLs from source to target cluster.
 `MirrorCheckpointConnector`:: (Optional) Synchronizes consumer group offsets for failover support. Only needed if offset tracking is required.
-`MirrorHeartbeatConnector`:: (Optional) Monitors connectivity between clusters. Useful for alerts but not required for replication. 
```

Comment:
Should we leave this listed here but note that it can't be configured via the KafkaMirrorMaker2 CR?